client/web/src/components/panels/Document.vue

```diff
@@ -328,25 +426,39 @@ export default defineComponent({
 			select_tool(toolName);
 		},
 		async viewModeChanged(toolIndex: number) {
-			console.log(`The view mode has been changed to index match the icon at index ${toolIndex}`);
-		},
+			console.log(
+				`The view mode has been changed to index match the icon at index ${toolIndex}`
+			);
+		}
 	},
 	mounted() {
-		registerResponseHandler(ResponseType.UpdateCanvas, (responseData) => {
-			this.viewportSvg = responseData.split("\n").map((shape, i) => shape.replace("#fff", `#${Math.floor(Math.abs(Math.sin(i + 1)) * 16777215).toString(16)}`)).join("\n");
+		registerResponseHandler(ResponseType.UpdateCanvas, responseData => {
+			this.viewportSvg = responseData
+				.split("\n")
+				.map((shape, i) =>
+					shape.replace(
+						"#fff",
+						`#${Math.floor(
+							Math.abs(Math.sin(i + 1)) * 16777215
+						).toString(16)}`
+					)
+				)
+				.join("\n");
 		});
-		registerResponseHandler(ResponseType.SetActiveTool, (responseData) => {
+		registerResponseHandler(ResponseType.SetActiveTool, responseData => {
```

Comment:
I feel like you probably have a local ESLint or autoformatter config which is making these changes for you, since our ESLint config apparently isn't set up correctly. That is something I have tried to fix but wasn't successful so far. We're standardizing on always including the `(parentheses)` in arrow functions.

Reply:
It seems that Vetur's prettier remove that kind of thing =/